src/components/Checkbox/docs/index.story.tsx

```diff
@@ -68,6 +68,30 @@ export default {
               source: examples.exampleWithIndeterminate,
             },
           ].map(code),
+          ...[
+            {
+              title: 'With box',
+              source: examples.exampleWithBox,
+            },
+          ].map(code),
+          ...[
+            {
+              title: 'With checked box',
+              source: examples.exampleWithCheckedBox,
+            },
+          ].map(code),
+          ...[
+            {
+              title: 'With disabled, checked box',
+              source: examples.exampleWithDisabledCheckedBox,
+            },
+          ].map(code),
+          ...[
+            {
+              title: 'With suffix',
+              source: examples.exampleWithSuffix,
+            },
+          ].map(code),
```

Comment:
I see that this code was written this way before your PR, but instead of having multiple 
```js
.map(code)
```

let's just have one array of examples, and one invocation of `map`.
ie
```js
...[
      {
        title: 'Checked',
        source: examples.exampleWithChecked,
      },
      ...
      {
        title: 'With box',
        source: examples.exampleWithBox,
      },
      ...
  ].map(code),
```

or even:
```js
const codeExamples = [
      {
        title: 'Checked',
        source: examples.exampleWithChecked,
      },
      ...
      {
        title: 'With box',
        source: examples.exampleWithBox,
      },
      ...
];

...

...codeExamples.map(code)
```